test/integration/1-system/inspec.yml

```diff
@@ -0,0 +1,13 @@
+name: system
+title: Integration Tests for base OS and its configuration
+summary: >
+  Tests for base OS, eg Ubuntu 16.04 LTS to verify that modifications and adjustments made
+  like user, packages, services and other Linux-related changes are applied and work as expected.
```

Comment:
Need to remove `1-system` alltogether, - these tests are Vagrant-specific for Packer.